gr-digital/lib/ofdm_carrier_allocator_cvc_impl.cc

```diff
@@ -73,6 +73,12 @@ namespace gr {
 	d_symbols_per_set(0),
 	d_output_is_shifted(output_is_shifted)
     {
+      // Sanity checks
+      // Since C++11: Get pointer to underlying array
+      // If that is is null, the input is wrong -> force user to use ((),) in python
+      if (d_occupied_carriers.data() == nullptr) {
```

Comment:
`nullptr` was introduced with C++11. Thus, this commit must go into next aka GR3.8.

Reply:
Is there no C++11 code at all up to 3.12.7?

Reply:
3.7.12, but yes: no C++11 until 3.8 due to compatibility with GCC 4.x which lacks reasonable C++11 support.